README.md

```diff
@@ -115,25 +139,46 @@ const worker: Worker = new MyWorker();
   <tbody>
     <tr>
       <td align="center">
-        <a href="https://github.com/sokra">
-          <img width="150" height="150" src="https://github.com/sokra.png?s=150">
+        <a href="https://github.com/TrySound">
+          <img width="150" height="150" src="https://avatars3.githubusercontent.com/u/5635476?v=3&s=150">
         </a>
         <br />
-        <a href="https://github.com/sokra">Tobias Koppers</a>
+        <a href="https://github.com/TrySound">Bogdan Chadkin</a>
+      </td>
+      <td align="center">
+        <a href="https://github.com/bebraw">
+          <img width="150" height="150" src="https://github.com/bebraw.png?v=3&s=150">
+          </br>
+          Juho Vepsäläinen
+        </a>
       </td>
       <td align="center">
         <a href="https://github.com/d3viant0ne">
-          <img width="150" height="150" src="https://avatars2.githubusercontent.com/u/8420490?v=3&s=150">
+          <img width="150" height="150" src="https://github.com/d3viant0ne.png?v=3&s=150">
+          </br>
+          Joshua Wiens
         </a>
-        <br />
-        <a href="https://github.com/d3viant0ne">Joshua Wiens</a>
       </td>
       <td align="center">
-        <a href="https://github.com/TrySound">
-          <img width="150" height="150" src="https://avatars3.githubusercontent.com/u/5635476?v=3&s=150">
+        <a href="https://github.com/sapegin">

```

Comment:
It's from the template, `Repo Maintainer(s)` + `Org Maintainers` agreed to slim it down a bit, but needs an update in ` defaults aswell then :)